src/Simplify/AstHelpers.elm

```diff
@@ -396,12 +396,12 @@ isIdentity lookupTable baseExpressionNode =
         Expression.FunctionOrValue _ "identity" ->
             ModuleNameLookupTable.moduleNameFor lookupTable expressionWithoutParensNode == Just [ "Basics" ]
 
-        Expression.LambdaExpression { args, expression } ->
-            case args of
+        Expression.LambdaExpression lambda ->
```

Comment:
Suggestion: Don't have commits named "Refactor X". Instead, say how you changed X. The commit message doesn't have to be great, but refactor is not super helpful/clear.

An additional tip (not to be applied all the time though): if your commit says "Did X and Y", you probably made too many changes at once. That's my way of working at least but to each their own :wink: